api/Document.json

```
@@ -1761,6 +1863,10 @@
               "safari_ios": {
                 "version_added": null
               },
+              "samsunginternet_android": {
+                "version_added": false,
```

I think this is true and Chrome Android is falsely marked as...well, `false`, considering Chrome, Opera Android, and WebView Android all have support.  I'm opening up a PR to resolve Chrome Android!

```suggestion
                "version_added": true,
```